### ISSUE TITLE
[KafkaSpinner] Attempt to get working on OSX

### ISSUE THREAD

**Issue by jeroldleslie** [DemandCube/NeverwinterDP_Legacy#384]

**Comment by KingOfPoptart** [comment]
http://viget.com/extend/how-to-use-docker-on-os-x-the-missing-guide
How to Use Docker on OS X: The Missing Guide | Viget
Have you heard of Docker? You probably have—everybody’s talking about it. It’s the new hotness. Even my dad’s like, “what’s Docker? I saw someone twitter about it on the Facebook. You should call your mom.”

tuan08 [8:54 AM]
try to see if you can understand :simple_smile:

tuan08 [8:57 AM]
basically you need to change $hostname to $USER@$(boot2docker ip)

tuan08 [8:57 AM]
$USER@$hostname
